interface/orders/orders_results.php

```diff
@@ -343,7 +343,7 @@ function validate(f) {
         $form_to_date = $form_from_date;
     }
 
-    $form_proc_type = formData('form_proc_type') + 0;
+    $form_proc_type = isset($_REQUEST['form_proc_type']) ? $_REQUEST['form_proc_type'] : 0;
```

Comment:
To keep the code the same, prob should add 0 to $_REQUEST['form_proc_type'] after the ?
(I think is done to force it to an int and also prevents sql-injection)